stable/prometheus-adapter/templates/external-metrics-cluster-role.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if .Values.rbac.create -}}
+{{- if .Values.rules.external }}
```

Comment:
Combine these two if statements with `and`

Reply:
Done.